modules/network-observability-operator-install.adoc

```diff
@@ -7,6 +7,10 @@
 = Installing the Network Observability Operator
 You can install the Network Observability Operator using the {product-title} web console Operator Hub. When you install the Operator, you also create the `FlowCollector` resource. You can set specifications in the web console when you create the  `FlowCollector`.  
 
+.Prerequisites
+
+* Installed Loki. It is recommended to install Loki using the link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.6], which allows the ability to specify a `network` tenant for Loki. The `network` tenant uses a specific tenant mode only implemented in version 5.6 of the Loki Operator. 
```

Comment:
Actually `openshift-network` mode create a Loki `network` tenant that you specify [at the end of the gateway url](https://github.com/openshift/openshift-docs/pull/55076/files#diff-6633f3bc44d16753be358ceccdbfc8dc3d26f5eb6da943e292437b6def959d07R24) in our CRD and provides automatic authentication and authorization in Openshift.

@stleerh what about removing the tenant mention here being more generic ?
```
It is recommended to install Loki using Loki Operator version 5.6, in which you should create a LokiStack instance using `openshift-network` configuration mode. It provides fully automatic in-cluster authentication and authorization support for Network Observability.
```

Reply:
@jpinsonneau @stleerh Okay, I see that in the create LokiStack procedure, `openshift-netowrk` is the specified mode. I guess this mention of `network` slipped.

Reply:
Either approach works. Maybe more generic is better.

Reply:
@jpinsonneau My suggested text says "tenant configuration mode" because the heading text that Loki Operator uses in the LokiStack form is "Tenants Configuration Mode".

@skrthomas The reason you had `network` is most likely because this is the way you configured it in Loki Operator 5.4 and 5.5.  You set the **tenant** field in FlowCollector to the value of `network`.  The **tenant** field is now ignored when using Loki Operator 5.6 in favor of using the tenant configuration mode.

Reply:
Okay thanks for elaborating on the detail there about "tenant configuration mode". I've updated the text accordingly